helm/aws-operator/values.yaml

```diff
@@ -1,4 +1,11 @@
 namespace: giantswarm
 
-userID: 1000
-groupID: 1000
+docker:
+  image:
+    tag: [[ .DockerTag ]]
+pod:
+  user:
+    id: 1000
+  group:
+    id: 1000
+version: [[ .Version ]]
```

Comment:
@xh3b4sd I moved all first level templating here.

Reply:
Thx IMO that is a nice improvement. One problem is I think this "draughtsman" templating breaks when you install the chart locally.

With this only the values.yaml needs to be changed. :+1: